packages/web-components/README.md

```diff
@@ -114,6 +114,44 @@ Once you do that, you can use our components as easy as using HTML tags, like:
 
 > 💡 Above CodeSandbox example uses [`html-webpack-plugin`](https://webpack.js.org/plugins/html-webpack-plugin/) to let [WebPack server](https://webpack.js.org/configuration/dev-server/) serve the `.html` file, but you can use other means to serve `.html` files, for example, using [Express](http://expressjs.com) server.
 
+### Using with legacy IBM.com Design System (Northstar)
+
+[Shadow DOM](https://w3c.github.io/webcomponents/spec/shadow/), one of the standards Carbon for IBM.com Web Components library is created on top of, isolates IBM.com Web Components library's component style from your application style, that means those two styles won't adversely affect each other.
```

Comment:
```suggestion
[Shadow DOM](https://w3c.github.io/webcomponents/spec/shadow/), one of the standards for Carbon for IBM.com Web Components is created on top of, isolates the web component styles from the application styles. This means those two styles won't adversely affect each other.
```

Reply:
Thank you for the suggestion @jeffchew - Incorporated.